themes/console-home/static/css/fonts.css

```diff
@@ -3,46 +3,46 @@
     font-family: "Rubik";
     font-style: normal;
     font-weight: 300;
-    src: url("../fonts/rubik-v11-latin-300.eot"); /* IE9 Compat Modes */
-    src: local(""), url("../fonts/rubik-v11-latin-300.eot?#iefix") format("embedded-opentype"), /* IE6-IE8 */ url("../fonts/rubik-v11-latin-300.woff2") format("woff2"), /* Super Modern Browsers */ url("../fonts/rubik-v11-latin-300.woff") format("woff"), /* Modern Browsers */ url("../fonts/rubik-v11-latin-300.ttf") format("truetype"), /* Safari, Android, iOS */ url("../fonts/rubik-v11-latin-300.svg#Rubik") format("svg"); /* Legacy iOS */
+    src: url("../fonts/rubik-v12-latin-300.eot"); /* IE9 Compat Modes */
```

Comment:
These font files don't exist in the branch - https://github.com/consoledotdev/home/tree/performance-tweaks/themes/console-home/static/fonts - they're all `v11` but you have `v12` in the pathname

Reply:
ah good catch, thanks, pushed the update